.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,68 @@
+  - jdk8 pipeline(s):
+    - primary jdk8 pipeline:
+      - rerun(s):
+    - secondary jdk8 pipeline: 
+      - reruns(s):
+  - jdk11 pipeline(s): 
+    - primary jdk11 pipeline:
+      - rerun(s):
+    - secondary jdk11 pipeline:
+      - rerun(s):
+  - jdkxx pipeline(s): 
+    - primary jdkxx pipeline:
+      - rerun(s):
+    - secondary jdkxx pipeline:
+      - rerun(s):
+- [ ] **Summarize test results**.  Find each launched build pipeline in [TRSS](https://trss.adoptium.net/) to view a summary of test results.  Can use the Release Summary Report feature in TRSS to generate a summary of failures, history and possible issues in markup format to be added to this issue as a comment. 
+- [ ] **Triage** each build and test failure in the release summary report (following the [Triage guidelines](https://github.com/adoptium/aqa-tests/blob/master/doc/Triage.md)) and determine blocking or non-blocking.  Supply links to triage issues or docs for each version here.
+  - jdk8 triage summary:
+  - jdk11 triage summary:
+  - jdkxx triage summary:
+- [ ] **Fix** blocking failures if they exist and confirm others are non-blocking.
+- [ ] **Get TSC 'ready to publish' approval**, once no blocking failures exist.
+- [ ] **Publish the release** (run the restricted access [release tool job](https://ci.adoptopenjdk.net/job/build-scripts/job/release/job/refactor_openjdk_release_tool/) on Jenkins)
+- [ ] **Verify binaries published successfully** to github releases repo and website (_automate_*, this could also be an automated test)
+- [ ] **Update support page** (_automate_* github workflow to create a PR to update [support.handlebars](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/src/handlebars/support.handlebars))
+- [ ] **Update release notes** (_automate_* - github workflow to create update for [release notes page](https://adoptium.net/release_notes.html))
+- [ ] **Run homebrew-cask_updater** (via [GitHub actions Updater job](https://github.com/AdoptOpenJDK/homebrew-openjdk/actions/workflows/updater.yml)) once binaries published successfully (this can be automated / triggered by a test for published artifacts)
+- [ ] **Trigger linux installers pipeline** currently it is part of the build pipelines (will eventually be updated to run independently)
+- [ ] **Publicize the release** via Slack #release channel and Twitter (can be partially automated)
+- [ ] **Trigger docker images pipeline** via the [GitHub Actions job](https://github.com/AdoptOpenJDK/openjdk-docker/actions/workflows/updater.yml). Click Run Workflow, keep the branch as Master and click `Run Workflow`. This updater job checks the API and updates the relevant dockerfiles. A Pull Request is then created by the adoptopenjdk-bot account which will need merging once all the CI tests pass.  *TBD - how this step evolves for Temurin releases.
```

Comment:
remove this please - adoptopenjdk specific

Reply:
~~ strike ~~

Reply:
~~strike~~